pipeline/tests/powdr_std.rs

```diff
@@ -193,12 +181,6 @@ fn bus_permutation_via_challenges_bn() {
     test_halo2(make_simple_prepared_pipeline(f));
 }
 
-#[test]
```

Comment:
Any chance we can keep them and add `#[should_panic = "error here"]` ?

Reply:
Most of these tests are redundant (see description). We could test that the "old" `test_data/std/lookup_via_challenges.asm` fails, but I don't really see a value in that, because it is far from any machine that would be used in practice IMO. We'll never have a solver that works in all cases, witgen will always need to make assumptions about the structure of the constraints.